packages/core/src/components/DismissableBanner/DismissableBanner.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, ReactNode, useState } from 'react';
+import classNames from 'classnames';
+import { makeStyles, Theme } from '@material-ui/core';
+import Snackbar from '@material-ui/core/Snackbar';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import IconButton from '@material-ui/core/IconButton';
+import Close from '@material-ui/icons/Close';
+// import { useSetting, Setting } from 'shared/apis/settings';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    position: 'relative',
```

Comment:
Is this and zIndex really needed?

Reply:
Turns out, yes, relative is needed. But not zIndex.

![Screenshot 2020-05-28 at 11 56 42](https://user-images.githubusercontent.com/190856/83127425-6e2d5200-a0da-11ea-93d5-e0e2242ddb4c.png)